md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -37,6 +37,8 @@ You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB
    * `org.bonitasoft.engine.api-type.server.url`: it is the url of the server, e.g. for a engine on the same host it can be `http://localhost:8080`
    * `org.bonitasoft.engine.api-type.application.name`: it is the name of the web aplication on wich the engine HTTP API is deployed, e.g. `bonita`
 
+  In addition the number of connections used by the client can be configured with system property `org.bonitasoft.engine.api-type.connections.max`, the default is 20.
```

Comment:
⚠️ I suggest to state this is only available as of Bonita 7.9.4
Otherwise people using 7.9.3 may think that this option is available